### DISCUSSION TITLE
WordPress Multisite - 50 Sub-Sites

### DISCUSSION THREAD

**Discussion by tovbar** [plausible/analytics#2091] [Anything Else]
Hello,
I have a question regarding the 50 websites.
I have a WordPress Multisite networks with over 50 sub-sites, which are all under one domain and one product.
For example:
xxx.com/sub-site1
xxx.com/sub-site2
xxx.com/sub-site3

I'm interested to use Plausible in order to collect analytics from each sub-site separately, and then display it in a dashboard in our main site (xxx.com). Right now we are using Koko Analytics successfully, but we're thinking to upgrade to your product.

Because all the sub-sites are under 1 domain and 1 product, I just want to confirm they will not count as separated websites so we don't have to worry about the 50 websites limitation per license.

Thanks! :)

**Comment by RobertJoonas** [comment]
Hi @tovbar!

Thanks for your interest in Plausible!

We have several features that support this type of use case:

1. Embed feature which allows you to embed our default dashboard (without Plausible branding) in your UI https://plausible.io/docs/embed-dashboard

2. Stats API that allows you to take individual metrics and create a custom-built dashboard within your UI https://plausible.io/docs/stats-api

3. Sites API that allows you to add and manage a large number of sites programmatically https://plausible.io/docs/sites-api

4. Proxy that allows you to run Plausible as a first-party connection from your domain name so there are no third-party calls and you get more accurate stats as adblockers don't block it https://plausible.io/docs/proxy/introduction.

If you want all your sub-sites to have a separate Plausible dashboard, the 50 sites limit will still apply. For accounts with more sites, we have enterprise plans where the cost depends on how many sites you have. Please feel free to reach out to our support at hello@plausible.io, so we can discuss your needs and figure out a suitable plan for you. Thanks!
